src/main/kotlin/uk/gov/dluhc/printapi/service/FilenameFactory.kt

```diff
@@ -22,5 +22,7 @@ class FilenameFactory(private val clock: Clock) {
 
     companion object {
         private var TIMESTAMP_FORMATTER: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyyMMddHHmmssSSS")
+
+        fun createFileNamePath(fileDirectory: String, fileName: String) = "$fileDirectory/$fileName"
```

Comment:
Is there any reason why this function is in the companion object (kinda like a java static) rather than an instance (bean) member? Just wondering why the change of approach ¯\_(ツ)_/¯